CODEOWNERS

```diff
@@ -389,6 +389,9 @@ tests/knative-build @rakesh-garimella @lilitgh @k15r @nachtmaar @montaro @marcob
 # Event Service Tests
 /tests/integration/event-service/ @Abd4llA @joek @abbi-gaurav @rakesh-garimella @lilitgh @k15r @nachtmaar @anishj0shi @montaro @marcobebway @radufa @sayanh @antoineco
 
+# Kafka Channel Integration Tests
+/tests/contract/knative-channel-kafka/ @Abd4llA @joek @abbi-gaurav @rakesh-garimella @lilitgh @k15r @nachtmaar @anishj0shi @montaro @marcobebway @radufa @sayanh @antoineco
```

Comment:
I think we can stop adding @abbi-gaurav and maybe @rakesh-garimella ?

Reply:
What about @joek?

Reply:
Yes I think it makes sense